test/k8sT/Updates.go

```diff
@@ -93,6 +89,13 @@ func InstallAndValidateCiliumUpgrades(kubectl *helpers.Kubectl, oldVersion, newV
 			oldVersion, helpers.GetCurrentK8SEnv()))
 		return func() {}, func() {}
 	}
+	switch helpers.GetCurrentIntegration() {
+	case helpers.CIIntegrationFlannel:
+		Skip(fmt.Sprintf(
```

Comment:
Will we be able to add upgrade / downgrade tests for Flannel support in Cilium v1.5? If so file a GH Issue and link it here so we don't miss that testing gap for v1.5.